src/styles/stories/colours.jsx

```diff
@@ -1,19 +1,95 @@
 import React from 'react'
 
-import { Swatch } from '../../index'
-import { GridLayout } from '../../util'
+import { ThemeProvider } from '../'
+import { Swatch, SwatchGroup } from '../../Swatch'
+import Typography from '../../Typography'
+
+import accentTheme from '../themes/accent'
+import awareTheme from '../themes/aware'
+import coreTheme from '../themes/core'
+import defaultTheme from '../themes/default'
+import neutralTheme from '../themes/neutral'
+import positiveTheme from '../themes/positive'
```

Comment:
Got the idea of breaking themes apart! Great way on working MUI's limitation on multiple colors :clap: 

I wonder how we could use this with other components, when importing `@theconversation/ui` while still wrapping them within the `ThemeProvider`. Example, if I want to create a "neutral" donation banner, I would like to use the following API:

```jsx
<ThemeProvider theme="neutral">
  <DonationBanner>
    The Conversation provides clean information in a world infected with spin. Please donate.
  </DonationBanner>
</ThemeProvider>
```

Right now, themes are not exported at the `src/index.js`, and IMHO, they shouldn't! (cc @nullobject @nickbrowne), but if they're are available for wider usage, they should be part of a configurable prop.

----

If this API gets through, and `theme` is a configurable prop with a subset, the `ThemeProvider` component should be able to handle it:

```js
// At the `src/styles/ThemeProvider.jsx` file propTypes definitions
theme: PropTypes.oneOf(['default', 'core', 'accent']), // and others....
```

Reply:
This makes each component responsible for wrapping _itself_ in a theme component?

The thinking with this was that we would leave the wrapping of a component in a theme to where it's used, alleviating the components themselves of having to know anything about what colours to use.

Reply:
Yeah, not the right call! @nullobject proposal is way better!